rootfs/scheduler/resources/pod.py

```diff
@@ -719,7 +719,41 @@ def _handle_not_ready_pods(self, namespace, labels):
                     message = "\n".join([x.strip() for x in event['message'].split("\n")])
                     raise KubeException(message)
 
-        return None
+        # Handle cases when no one pods is started (quota check fialed)
+        # If quota is enabled in a namespace for compute resources like cpu and
+        # memory, users must specify requests or limits for those values;
+        # otherwise, the quota system may reject pod creation.
+        # http://kubernetes.io/docs/admin/resourcequota/
+        # or when scale process trigger overuse case for some pods:
+        # For example: used 3/3 pods, but user scale to 4
+        events = self._get_namespace_events(namespace)
+        if len(events) != 0:
+            log = self._get_formatted_messages(events)
+            self.log(namespace, log)
+            raise KubeException(log)
```

Comment:
> Can you check specifically for events that fail a quota check instead?
It is my first plan, but it is very hard to detect quota check events, because events about quota not linked with any pods.